int64 id
int64[] region_id
string group_name
---
bool done
#puntos en regiones a visitar
ltlplanner/position[] positions
---
bool done
